src/components/App/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import logo from '../../logo.svg';
+import './App.css';
+import MainApp from '../MainApp';
+import {
+    useSelector,
+} from 'react-redux';
+
+function App() {
+    const todos = useSelector((state: {list: { todos: any[] }}) => state.list.todos);
```

Comment:
add a correct type instead of `any[]` - use `Todo[]` type (defined inside `src/MainApp/index.tsx`)